Adyen/UI/Form/Items/Address/AddressViewModel.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal enum AddressFields: String, CaseIterable {
```

Comment:
`AddressField` singular makes more sense to me than a plural :-)

Reply:
Good one! 👍